crates/corrosion/src/command/consul/sync.rs

```diff
@@ -705,7 +768,15 @@ mod tests {
             assert_eq!(app_id, 123);
         }
 
-        let (applied, _check_applied) = execute("node-1", &ta1_client, update_services(HashMap::new(), &svc_hashes, false), &mut svc_hashes, Default::default(), &mut check_hashes).await?;
+        let (applied, _check_applied) = execute(
+            "node-1",
+            &ta1_client,
+            update_services(HashMap::new(), &svc_hashes, false),
+            &mut svc_hashes,
+            Default::default(),
+            &mut check_hashes,
+        )
+        .await?;
 
         assert!(check_applied.is_zero());
 
```

Comment:
Do I need to review this file or is it just formatting? It's fine to include this, but I was wondering 😅

Reply:
I think this file is just formatting :sweat_smile:. I haven't figured out why format on save is slow on IDEA so I just run `cargo fmt` before I push